core/src/main/java/lucee/runtime/engine/ThreadLocalPageContext.java

```diff
@@ -47,6 +47,14 @@ public final class ThreadLocalPageContext {
 	private static ThreadLocal<Boolean> insideGateway = new ThreadLocal<Boolean>();
 	private static ThreadLocal<Boolean> insideInheritableRegistration = new ThreadLocal<Boolean>();
 
+	/**
+	 * Global kill switch for precise math.
+	 * - null: normal behavior, per-application/config control
+	 * - "disabled": globally disabled, throws error if anyone tries to enable it
+	 */
+	private static final String PRECISE_MATH_GLOBAL = SystemUtil.getSystemPropOrEnvVar( "lucee.precise.math", null );
+	public static final boolean PRECISE_MATH_GLOBALLY_DISABLED = "disabled".equalsIgnoreCase( PRECISE_MATH_GLOBAL );
+
```

Comment:
same her, just one and a boolean and not "disabled", this is also the wrong place to do this. ThreadLocal classes should only be used for thread local related stuff.